plone-5/instance/src/ukstats.ccv2.content_types/src/ukstats/ccv2/content_types/api/controlpanel.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.restapi.services import Service
+from plone.restapi.serializer.converters import json_compatible
+
+reg_base = 'cmsconf'
+
+
+class SiteTitle(Service):
+    def reply(self):
+        registry_record_value = api.portal.get_registry_record(reg_base + '.site_title')
```

Comment:
so the signature for `get_registry_record()` is:

`get_registry_record(name=None, interface=None, default=MISSING)` this means we're passing in a keyword argument positionally.

this works.... right up until someone changes the signature and suddenly we're populated the wrong parameter and everything breaks for hard to debug reasons (I've had to unfudge that more than once, pet hate).

willing to bet this came from a pone example not you, but don't do it, it's a shirt sighted thing that bad python developers like to do. Always use keywords as intended, python tip of the day 😄 .